src/App.tsx

```diff
@@ -37,27 +27,7 @@ export const App: React.VFC = () => {
                 }}
             >
                 <Header />
-                <Box m={2} my={4}>
-                    <Paper sx={{ px: 2, py: 0.5 }}>
-                        <Typography variant="body2">
-                            {meta.published}
-                        </Typography>
-                        <Typography variant="h6">{meta.title}</Typography>
-                        {open ? (
-                            <Box mt={0.5}>
-                                <Divider />
-                                <Example />
-                                <Link onClick={() => setOpen(false)}>
-                                    &raquo; close
-                                </Link>
-                            </Box>
-                        ) : (
-                            <Link onClick={() => setOpen(true)}>
-                                &raquo; read more
-                            </Link>
-                        )}
-                    </Paper>
-                </Box>
+                <Outlet />
```

Comment:
This is a new feature of React Router v6, `<Outlet />` renders a React component for child route.